ptx/S_12_6_Curl.ptx

```diff
@@ -5,65 +5,315 @@
+        <p>
+          We would like to measure and understand rotation at the local level for a vector field. In order to investigate this concept, we will look at some two dimensional vector fields and think about whether the vector field shown will rotate a small pinwheel or spinner placed at a particular location.
         </p>
+        <figure xml:id="spinner_fig">
+          <caption> <p>A picture of our model spinner</p></caption>
+          <image source="images/spinner_gif.gif"/>
+        </figure>
+      </introduction>
+        <task xml:id="Curl_PA_task1">
+          <p> For this part, look at the vector field in <xref ref="Curl_PA_task1_vf">Figure</xref>.</p>
+          <figure xml:id="Curl_PA_task1_vf">
+            <image source="images/Curl_-yx_field"/>
+          </figure>
+            <ol>
+              <li>
+                <p>Would a small spinner placed at the origin spin clockwise, counterclockwise, or not spin at all?</p>
+              </li>
+              <li>
+                <p>Would a small spinner placed at <m>(1,1)</m> spin clockwise, counterclockwise, or not spin at all?</p>
+              </li>
+              <li>
+                <p>Would a small spinner placed at <m>(-2,1)</m> spin clockwise, counterclockwise, or not spin at all?</p>
+              </li>
+               <li>
+                <p>Are there any points on the <m>xy</m>-plane where the spinner would not turn counterclockwise?</p>
+              </li>
+            </ol>
         </task>
         <task xml:id="Curl_PA_task2">
-          <p> show a picture of (x,y) and ask about whether the spinner would go clockwise, counterclockwise, or not spin if placed at (0,0), (1,1), (-2,-1)
-          </p>
+          <p> For this part, look at the vector field in <xref ref="Curl_PA_task2_vf">Figure</xref>.</p>
+          <figure xml:id="Curl_PA_task2_vf">
+            <image source="images/Curl_xy_field"/>
+          </figure>
+            <ol>
+              <li>
+                <p>Would a small spinner placed at the origin spin clockwise, counterclockwise, or not spin at all?</p>
+              </li>
+              <li>
+                <p>Would a small spinner placed at <m>(1,1)</m> spin clockwise, counterclockwise, or not spin at all?</p>
+              </li>
+              <li>
+                <p>Would a small spinner placed at <m>(-2,1)</m> spin clockwise, counterclockwise, or not spin at all?</p>
+              </li>
+               <li>
+                <p>Are there any points on the <m>xy</m>-plane where the spinner would turn differently than your descriptions in the previous part of this problem?</p>
+              </li>
+            </ol>
         </task>
         <task xml:id="Curl_PA_task3">
-          <p> show a picture of (2,1), etc.</p>
+          <p> For this part, look at the vector field in <xref ref="Curl_PA_task3_vf">Figure</xref>.</p>
+          <figure xml:id="Curl_PA_task3_vf">
+            <image source="images/Curl_12_field"/>
```

Comment:
Several of the `image` are missing `@width`, so HTML has really, really big images. This is not a deal breaker right now, as we may want to try to pick a uniformly appealing width for as many of them as possible later on.